Pipelines/Templates/Package.yaml

```diff
@@ -9,7 +9,7 @@ jobs:
   displayName: Generate Unity and NPM Packages
   pool:
     name: Analog On-Prem
-    demands: Unity2019LTS
+    demands: Unity2020LTS

```

Comment:
I thought I remember Cierra saying demands don't exist anymore

Reply:
No sure what happened to my comments. :/ Going to leave this in for now since at worst it's ignored. I'll remove it if it's not needed when pulling feature into develop.